### PR TITLE
TC-128: Sends an email notification to a specific list of contest man…

### PR DESCRIPTION
**Pull request by natebessa**
…agers upon team finalization

Adds a SITE_URL setting to make it easier to get the full URL to the application.

### CODE REVIEW

File: app/contact/views.py [hunk 1]
```diff
@@ -39,9 +39,9 @@ def contact_form(request):
             recipients = settings.CONTACT_FORM_RECIPIENTS.split(',')
 
             # Send it out.
-            success = email_send(subject='PPM Contact Form Inquiry',
+            success = email_send(subject='Hypatio Contact Form Inquiry',
```
Comment (mtmcduffie) [inline app/contact/views.py]: Can you change this to DBMI Data Portal instead of Hypatio?

File: app/contact/views.py [hunk 2]
```diff
@@ -83,32 +83,29 @@ def contact_form(request):
...
-        sent_without_error = True
+    sent_without_error = True
 
-        extra["user_email"] = r
+    msg_html = render_to_string('email/%s.html' % email_template, extra)
+    msg_plain = render_to_string('email/%s.txt' % email_template, extra)
 
-        msg_html = render_to_string('email/%s.html' % message, extra)
-        msg_plain = render_to_string('email/%s.txt' % message, extra)
+    logger.debug("[P2M2][DEBUG][email_send] About to send e-mail.")
```
Comment (mtmcduffie) [inline app/contact/views.py]: Hypatio here

File: app/contact/views.py (hunk 1 above)
Comment (natebessa) [reply app/contact/views.py]: Derp -- right, will do.

**Comment by natebessa** [comment]
@mtmcduffie all set

**Pull request merged**